conf/gunicorn-dev.conf.py

```diff
@@ -2,8 +2,10 @@
 
 
 bind = "0.0.0.0:5000"
-graceful_timeout = 0
```

Comment:
It seems to be enough to set `timeout = 0`, I don't think `graceful_timeout` does anything when `timeout = 0`